Gateway/Response/AuthorizeTransactionIdHandler.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Buckaroo\Magento2\Gateway\Response;
+
+use Buckaroo\Transaction\Response\TransactionResponse;
+use Magento\Sales\Api\Data\OrderPaymentInterface;
+use Buckaroo\Magento2\Gateway\Helper\SubjectReader;
+use Magento\Payment\Gateway\Response\HandlerInterface;
+use Magento\Sales\Model\Order\Payment;
+
+class AuthorizeTransactionIdHandler extends TransactionIdHandler
```

Comment:
what's the use of this class?

Reply:
When we make an order with KlarnaKP we shouldn't close transaction. app/code/Buckaroo/Magento2/Model/Method/AbstractMethod.php:938

![image](https://user-images.githubusercontent.com/106905746/220155783-0b154145-a4d1-4fff-b161-6ca5a5514b67.png)